apps/etterlatte-utbetaling/src/main/kotlin/utbetaling/avstemming/GrensesnittsavstemmingJob.kt

```diff
@@ -52,12 +55,13 @@ class GrensesnittsavstemmingJob(
             withLogContext {
                 if (leaderElection.isLeader()) {
                     Saktype.values().forEach {
```

Comment:
Det er sikkert litt premature optimization å vurdere dette, men: Burde konsistensavstemmingene kjøres litt offsettet fra hverandre? Siden det er en del som gjøres opp mot både vår base og oppdrag i en gjennomkjøring. Men igjen, det er sikkert overkill å tenke for mye på det :D